libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/xirr/tests/PluggableXirr.test.ts

```diff
@@ -100,22 +91,12 @@ describe("PluggableXirr", () => {
 
             xirr.update(options, testMocks.insightWithSingleMeasure, emptyPropertiesMeta, executionFactory);
 
-            expect(reactCreateElementSpy.mock.calls[0][0]).toBe(CoreXirr);
-            expect(mockRenderFun).toHaveBeenCalledWith(
-                fakeElement,
-                document.querySelector(defaultProps.element),
-            );
-
-            reactCreateElementSpy.mockReset();
+            const renderEl = getLastRenderEl<ICoreChartProps>(mockRenderFun, mockElement);
+            expect(renderEl.type).toBe(CoreXirr);
+            expect(renderEl.props.config.enableCompactSize).toBeUndefined();
```

Comment:
I added an explicit checks for the FF here and in a few other places. Not sure if this is not too much, the test should not know about implementation details. What is our position on this?

Reply:
It is a bit complicated, we prefer black-box testing, but historically, many of the enzyme tests were heavily white-box. We plan to get rid of enzyme altogether as it is basically dead and use testing-library which does not even support assertions like this.

So TLDR don't worry too much about it, the enzyme is going away anyway.